app/themes/default/views/layouts/default.html.slim

```diff
@@ -33,7 +33,18 @@ html
         div.collapse.navbar-collapse#site-navigation-navbar
           ul.nav.navbar-nav.pull-right
             li
-              = link_to(t('layout.navbar.courses'), courses_path)
+              - my_courses = user_signed_in? && Course.with_approved_user(current_user).ordered_by_end_at
+              - if my_courses.present?
+                a.dropdown-toggle data-toggle="dropdown"
+                  => t('layout.navbar.courses')
+                  span.caret
+                ul.dropdown-menu.pull-right
+                  - my_courses.each do |course|
+                    li = link_to(format_inline_text(course.title), course_path(course))
+                  li.divider role='separator'
+                  li = link_to(t('layout.navbar.all_courses'), courses_path)
+              - else
+                = link_to(t('layout.navbar.courses'), courses_path)
```

Comment:
# Signed In
<img width="559" alt="screen shot 2016-06-22 at 7 47 37 pm" src="https://cloud.githubusercontent.com/assets/6252919/16267719/d7abad14-38bd-11e6-8281-9f7016c58a3e.png">

# Not Signed In
<img width="369" alt="screen shot 2016-06-22 at 9 09 22 pm" src="https://cloud.githubusercontent.com/assets/6252919/16267720/d7d0f27c-38bd-11e6-98cb-5d2aed9cc040.png">

@weiqingtoh seems to be suggesting
(A) Course
(B) All Courses
(C) All Courses

@allenwq Are you suggesting the following?:
(A) My Course
(B) All Courses
(C) All Courses